src/main/java/io/cdap/plugin/servicenow/connector/ServiceNowConnectorConfig.java

```diff
@@ -136,22 +155,62 @@ public void validateConnection(FailureCollector collector) {
     }
   }
 
-  public Object convertToValue(String fieldName, Schema fieldSchema, Map<String, Object> record) {
-    Schema.Type fieldType = fieldSchema.getType();
+  public void convertToValue(String fieldName, Schema fieldSchema, Map<String, Object> record,
+                               StructuredRecord.Builder recordBuilder) {
     Object fieldValue = record.get(fieldName);
+    if (fieldValue == null || fieldValue.toString().isEmpty()) {
```

Comment:
this `toString().isEmpty()` check is odd. When would that be empty instead of null, and when would we ever want to do this for a non-string object?

Reply:
ServiceNow sometimes returns blank or empty values instead of `null`. As all values are string, therefore remove the call to `toString()` method. a963c57e661447a3f1a6e970cdabc6f76964b313